book.toml

```diff
@@ -13,3 +13,42 @@ command = "mdbook-mermaid"
 
 [output.html]
 additional-js = ["doc/mermaid.min.js", "doc/mermaid-init.js"]
+
+[output.linkcheck]
+# Should we check links on the internet? Enabling this option adds a
+# non-negligible performance impact
+follow-web-links = false
```

Comment:
This would actually be a great feature.

Reply:
I agree, but this needs some more treatment to make it work, its quite sensitive.